js/private/translate_pnpm_lock.bzl

```diff
@@ -134,6 +136,10 @@ _ATTRS = {
     "no_optional": attr.bool(
         doc = """If true, optionalDependencies are not installed""",
     ),
+    "no_lifecycle_hooks": attr.string_list(
```

Comment:
starting with "no" makes it sound boolean.

`lifecycle_hooks_exclude` maybe?

Reply:
👍